### PR TITLE
Add new api

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| License       | MIT
| Packagist     |

### CODE REVIEW

**Comment by OskarStark** [comment]
@Nyholm can you please merge? I use null instead of `~` everywhere.

Error is not related, because SF 4 is not supported.

Thanks

File: datana-gmbh/datapool-api/1.0/config/packages/datana_datapool_api.yaml [hunk 1]
```diff
@@ -8,26 +8,32 @@ services:
...
+    Datana\Datapool\Api\AktenEventLogApi: null
+    Datana\Datapool\Api\NullAktenEventLogApi: null
 
-    Datana\Datapool\Api\AktenzeichenApi: ~
-    Datana\Datapool\Api\FakeAktenzeichenApi: ~
+    Datana\Datapool\Api\AktenzeichenApi: null
+    Datana\Datapool\Api\FakeAktenzeichenApi: null
+
+    Datana\Datapool\Api\ChatProtocolApi: null
+    Datana\Datapool\Api\NullChatProtocol: null
```
Comment (silasjoisten) [inline datana-gmbh/datapool-api/1.0/config/packages/datana_datapool_api.yaml:21]:
```suggestion
    Datana\Datapool\Api\NullChatProtocolApi: null  
```

Comment (silasjoisten) [reply datana-gmbh/datapool-api/1.0/config/packages/datana_datapool_api.yaml:21]: @OskarStark cc

Comment (OskarStark) [reply datana-gmbh/datapool-api/1.0/config/packages/datana_datapool_api.yaml]: Nice catch, thanks!

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**